services/core-api/app/api/now_submissions/namespace.py

```diff
@@ -16,3 +17,6 @@
                  '/applications/<string:application_guid>/document/<int:id>')
 api.add_resource(ApplicationDocumentTokenResource,
                  '/applications/<string:application_guid>/document/<int:id>/token')
+api.add_resource(ApplicationStatusResource, '/applications/<string:messageid>/status')
+api.add_resource(ApplicationStatusUpdatesSinceResource,
```

Comment:
Can this be changed to use a query parameter? Eg. /status?since=2020-04-04

Reply:
Definitely! I was expecting to make this change.